broadcasting/11_Summary.md

```diff
@@ -0,0 +1,7 @@
+As you can see converting an RTMP or RTSP stream to HLS might not be the easiest task, however, some understanding of the protocols and the usage of a couple of prefabricated elements in the Membrane Framework make up a rather compact solution, which is open for extension and modification.
+
+Being one of the most popular media streaming protocols, you will often come across HLS in the media streaming industry, so it's useful to have some understanding of it.
```

Comment:
I believe that chapter requires to be slightly refined - I would extend that paragraph which talks about why it is good to know HLS

Reply:
done